views/topic/edit-topic.hbs

```diff
@@ -44,10 +44,9 @@
 
                 <div class="form-group">
                     <label>{{$t "topic._topic.label.content" }}</label>
-
                     <!-- React Magic -->
-                    <div id="content-blocks" data-value="{{lesson.contents}}" data-parent-id="{{lesson._id}}" data-school-id="{{schoolId}}" data-parent-type="lessons" data-etherpadbaseurl="{{etherpadBaseUrl}}"
-                        data-iscoursegroup="{{courseGroupId}}" data-nexboardenabled="{{#ifEnv "FEATURE_NEXBOARD_ENABLED" "true"}}true{{/ifEnv}}"></div>
+                    <div id="content-blocks" data-testid="nexboardButton" data-value="{{lesson.contents}}" data-parent-id="{{lesson._id}}" data-school-id="{{schoolId}}" data-parent-type="lessons" data-etherpadbaseurl="{{etherpadBaseUrl}}"
```

Comment:
Why did you add the `data-testid` here? As I wrote in a comment before, I'm not an expert on this topic (should be verified by someone else), but doesn't it mean that the whole div will be treated as a **nexboardButton** and this is **not** a single button, but the container for all these buttons? @VikDavydiuk do you maybe know whether I'm right or not?

Reply:
@Michaellinaresxk why did you mark the conversation as resolved? Did you consult it with @VikDavydiuk or someone else? Please help me understand why did you add this `data-testid` param here.